schema.graphql

```diff
@@ -55,3 +55,15 @@ type ValidatorEpochInfo @entity {
     validatorId: String!
     epochUnstakedAmount: BigInt!
 }
+
+type EpochCleanup @entity {
+    """
+    The ID is the epoch ID
+    """
+    id: ID!
+
```

Comment:
remove empty lines?

Reply:
I deliberately added these empty lines for readability. Do they cause downsides?

Reply:
format not consistent with other schema definition